### DISCUSSION TITLE
2FA is the last nail in the coffin

### DISCUSSION THREAD

**Discussion by FrozenVoid** [community/community#63794] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

Its not that i lost access to Github for few years because of email verification, but
now GH is spiraling down to mobile walled garden social network, time for some alternatives.

**Comment by ghost** [comment, marked as answer]
Due to Github requiring 2FA on my hobby projects I've deleted all my repos and will abandon the platform.
I get it for big open source projects and such, but I'm a hobby programmer. I don't care about my profile security and I'm not giving GitHub any other information then they already have about me.
It should've been an **opt-in** feature for profiles that need it.

**Reply by Yoshiyuka** [reply]
I'm pissed that I'm being forced to use that 2FA garbage with my hobby projects as well. Done with github.

**Comment by phikal** [comment]
FWIW it seems that you can use a Firefox extension like [Authenticator](https://addons.mozilla.org/en-US/firefox/addon/auth-helper/) to authenticate.  Not great, but something.
